src/data/artworks/next_1000_cards.ts

```diff
@@ -0,0 +1,58 @@
+import type { ArtworkDescriptor } from '@/core/Artwork'
+
+export const next_1000_cards_artworks: ArtworkDescriptor[] = [
+  {
+    id: '2024-09-23_Card0003',
+    title: 'Card 3/1000',
+    date: '2024-09-23',
+    timeline_desc: 'Yet another doodle.',
+    sort_key: '2024-09-23:01',
+    project_id: 'next-1000-cards',
+    img_format: 'jpg',
+  },
+  {
+    id: '2024-10-02_Card0012',
+    title: 'Card 12/1000',
+    date: '2024-10-12',
+    timeline_desc: `
+        I was reading <cite><a href="https://www.google.com/books/edition/The_Algorithmic_Beauty_of_Plants/4F7lBwAAQBAJ?hl=en&gbpv=0">The Algorithmic Beauty of Plants</a></cite>
+        around this time, which explains the tree-shaped motifs.
+    `,
+    sort_key: '2024-10-02:01',
+    project_id: 'next-1000-cards',
+    img_format: 'jpg',
+  },
+  {
+    id: '2024-10-12_Card0019',
+    title: 'Card 19/1000',
+    date: '2024-10-12',
+    timeline_desc: `
+        This doodle inspired the "nacho" fractal I made in 
+        <a href="#/project/math-notebook">Math Notebook</a>. I was trying to
+        match the general structure of the classic
+        <a href="https://en.wikipedia.org/wiki/Sierpi%C5%84ski_triangle">Sierpiński triangle</a>
+        fractal, but using curved triangles instead.
+    `,
+    sort_key: '2024-10-12:01',
+    project_id: 'next-1000-cards',
+    img_format: 'jpg',
+  },
+  {
+    id: '2024-10-18_Card0027',
+    title: 'Card 27/1000',
+    date: '2024-10-18',
+    timeline_desc: 'Yet another doodle.',
+    sort_key: '2024-10-18:01',
+    project_id: 'next-1000-cards',
+    img_format: 'jpg',
+  },
+  {
+    id: '2024-11-03_Card0044',
```

Comment:
I just realized something... in October I drew a LOT! enough that I could say I accidentally did Inktober even though I wasn't trying to. That would be an amusing project update to include.

Reply:
I checked my binder. on 10-01 I drew card 11. On 10-31 I drew card 42. That's $$42 - 11 + 1 = 32$$ cards!